apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: Automatically Stop and Start App Machines
+objective: 
+layout: docs
+nav: firecracker
+order: 65
+---
+
+<%= partial "/docs/partials/v2_transition_banner" %>
+
+You can decide whether you want to automatically stop, or downscale, instances of your app when they're idle and automatically restart them when they're needed. Each instance of your app is running on a Fly machine. Fly Proxy controls automatically starting and automaticaly stopping machines according to settings in your `fly.toml` file, in either the `services` or `http_service` sections:
+
+```toml
+...
+[[services]]
+  autostart = true
+  autostop = true
+...
+```
+
+```toml
+...
+[http_service]
+  internal_port = 8080
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+...
+```
+
+## Default and recomended values
+
+New V2 apps are automatically started and automatically stopped by default:
+
+```toml
+auto_stop_machines = true
+auto_start_machines = true
+```
+
+Existing apps are automatically started but not automatically stopped by default:
+
+```toml
+auto_stop_machines = false
+auto_start_machines = true
+```
+
+In most cases, we recommend setting `auto_stop_machines` and `auto_start_machines` to the same value to avoid having instances that either never start or never stop. 
+
+If `auto_start_machines = true` and `auto_stop_machines = false`, the proxy will start your instances but they will never be stopped automatically. If you have a reason to only stop instances manually, then these settings are just fine.
+
+If `auto_start_machines = false` and `auto_stop_machines = true`, the proxy will scale your instances down but won't be able to start them again. If all of your instances are scaled down, then requests will start failing.
+
+## How It Works
+
+The Fly Proxy runs the process to stop and start machines automatically every few minutes.
+
+When `auto_stop_machines = true` in your `fly.toml`, the proxy looks at machines running in a single region and uses the [`soft_limit` setting](/docs/reference/configuration/#the-http_service-section) for each machine to determine if there's excess capacity. If the proxy decides there's excess capcity, it will stop exactly one machine. The proxy repeats this process every few minutes, stopping only one machine per region, if needed, each time.
+
+Fly proxy considers all of the running instances of your app in a given region, such as `fra`, and determines excess capacity as follows:
```

Comment:
Fly proxy -> Fly Proxy, unless we want to call it fly-proxy like its repository — or something else — a branding decision. I’ve been calling it Fly Proxy.

Reply:
I was mulling that. I don't have a perfect answer. "Our proxy" can also be handy? Since there are a few things "the proxy" could mean depending on the context, we just have to keep an eye out for any ambiguity.

If it's clear from the start that this whole feature is a feature of Fly Proxy, then it hopefully sets people up to think about our proxy first and not some other kind.